pyproject.toml

```
@@ -191,5 +191,5 @@ testpaths = [
 ]
 
 [build-system]
-requires = ["poetry>=1.0.0"]
-build-backend = "poetry.masonry.api"
+requires = ["poetry-core>=1.0.0"]
```

The version constraint is redundant. There has never been anything older.

```suggestion
requires = ["poetry-core"]
```